src/sentry/static/sentry/app/components/forms/form.jsx

```diff
@@ -31,6 +32,7 @@ const Form = React.createClass({
           <button className="btn btn-primary"
                   disabled={this.props.submitDisabled}
                   type="submit">{this.props.submitLabel}</button>
+          {this.props.extraButton}
```

Comment:
this just makes something like this possible:
![screenshot 2016-09-21 17 26 03](https://cloud.githubusercontent.com/assets/5026776/18733519/86009fe2-8020-11e6-84ad-148b48abc5cd.png)